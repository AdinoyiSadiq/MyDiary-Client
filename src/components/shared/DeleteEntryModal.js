import React from 'react';

const DeleteEntryModal = (props) => {
  const { display, hideDeleteModal, deleteEntry } = props;
  return (
  <section className={`authContent deleteModal ${ display ? 'displayModal' : null}`}>
		<div className="title">Do you want to delete this entry?</div>
			<hr/>
			<div className="clearfix">
        <button 
          id="deleteButton"
          onClick={deleteEntry}
        >
          Yes
        </button>
        <button 
          id="cancelButton" 
          onClick={hideDeleteModal}
        >
          No
        </button>
			</div>
	</section>
  )
};

export default DeleteEntryModal;