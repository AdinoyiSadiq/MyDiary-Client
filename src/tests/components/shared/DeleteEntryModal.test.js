import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import DeleteEntryModal from '../../../components/shared/DeleteEntryModal';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<DeleteEntryModal />);
});

describe('DeleteEntryModal UI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});