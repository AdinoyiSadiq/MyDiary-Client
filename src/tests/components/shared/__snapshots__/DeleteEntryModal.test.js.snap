// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteEntryModal UI render features view should render as expected 1`] = `
<section
  className="authContent deleteModal null"
>
  <div
    className="title"
  >
    Do you want to delete this entry?
  </div>
  <hr />
  <div
    className="clearfix"
  >
    <button
      id="deleteButton"
    >
      Yes
    </button>
    <button
      id="cancelButton"
    >
      No
    </button>
  </div>
</section>
`;
